js/render.js

```diff
@@ -0,0 +1,138 @@
+const Renderer = function () {
+    const container = $('#content-container')
+
+    const createElement = function (type, class1, text) {
```

Comment:
try to find another parameter name for `class1`. It does not describe exactly what the function will be receiving.